tests/page/page-aria-snapshot-ai.spec.ts

```diff
@@ -354,3 +354,23 @@ it('should mark iframe as active when it contains focused element', async ({ pag
         - textbox "Input in iframe" [active] [ref=f1e2]
   `);
 });
+
+it('return empty snapshot when iframe is not loaded', { annotation: { type: 'issue', description: 'https://github.com/microsoft/playwright/pull/36710' } }, async ({ page, server }) => {
+  await page.setContent(`
+    <div style="height: 5000px;">Test</div>
+    <iframe loading="lazy" src="${server.PREFIX}/frame.html"></iframe>
+  `);
+
+  // Wait for the iframe to load
+  await page.waitForSelector('iframe');
+
+  // Get the snapshot of the page
+  const snapshot = await snapshotForAI(page, { timeout: 100 });
```

Comment:
I think this test will fail because Playwright tests have a default timeout of 30 seconds. You might want to set the timeout value to less than 30 seconds.

Reply:
This is 100ms, so there's enough time.